test/helpers/StubXcmUtils.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Would be nice to have a bit more explanation about what this helper contract is doing for ppl unfamiliar with the codebase.